ckan/cli/__init__.py

```diff
@@ -0,0 +1,60 @@
+# encoding: utf-8
+
+import os
+
+import click
+import logging
+from flask import Flask, current_app
+from flask.cli import AppGroup, with_appcontext
+from werkzeug.serving import run_simple
+
+from ckan.common import config
+from ckan.config.environment import load_environment
+from ckan.config.middleware import make_app
+
+
+log = logging.getLogger(__name__)
+
+
+click_config_option = click.option(
+    u'-c',
+    u'--config',
+    default=None,
+    metavar=u'CONFIG',
+    help=u'Config file to use (default: development.ini)')
+
+
+def _load_config(config=None):
```

Comment:
This looks reusable, how about dropping the leading `_`?

There are parts of this function shared with the `_get_config` code in lib/cli.py, can they be factored out instead of repeated?

Reply:
im keeping the same code as it is in `_get_config` in `ckan/lib/cli.py`, but ive renamed it to `load_config`,  because we are still using pylons and we should call `make_app` to create both stacks.